JetBrains IDEs/README.md

```diff
@@ -0,0 +1,54 @@
+# JetBrains IDEs
+JetBrains is a company creating tools for developers. Their mission is: _We make professional software development a more productive and enjoyable experience_. In this section a number of IDEs (integrated development environment) as well as a selection of features will be presented.
+
+## Selection of Features
+**Note that all of these features apply to _all_ available IDEs.**
+
+### Shared Key Mappings
+If you use different programming languages you might also use different IDEs/editors. This forces you to learn different key mappings for each individual IDE/editor. With IDEs from JetBrains you only need to learn one key mapping because it's shared between all of their IDEs. If you customize your key mapping you can even export it from one IDE and import in the other one.
+
+### Highly Customizable
+Not only can you do lots of customization when it comes to appearance, key mapping and a lot more, you also have plugins. These plugins cover everything from dealing with different file types to using external tools such as Git, Docker, databases, linters and time tracking. JetBrains offer a bunch of their own plugins but you can also develop and publish your own plugins.
+
+### Integrated Tools
+Their IDEs offer so many tools that you never have to leave it if you don't want to. It supports a number of different VCSs (Git, Subversion, Perforce etc.), a terminal window, database browser, debuggers, SSH sessions and more.
+
+### Powerful Refactoring
+Rename a method used all over your project with a few clicks, or change a method signature in the same way. With statically typed languages like Java and C the refactoring becomes even more powerful.
+
+## Useful Shortcuts
+* Search for a file -- `CMD + SHIFT + O`
+* Search for a string -- `CMD + SHIFT + F`
+* Search everything, press `SHIFT` three (3) times
```

Comment:
Not sure, but on my PHPStorm installation I use `SHIFT` twice for search everywhere, maybe defaults changed? Or perhaps it differs from JetBrains product.

Reply:
I tried in a few different and yes, you are right. I'll change it.